twitter-user-feed.js

```diff
@@ -1,6 +1,7 @@
 var H5P = H5P || {};
 
 H5P.TwitterUserFeed = (function ($) {
+  H5P.EventDispatcher.call(this);
```

Comment:
This is probably enough for the current usage, but when "inherriting" the EventDispatcher we always include it in the prototype chain as well - something like this:

// Inheritance
C.prototype = Object.create(EventDispatcher.prototype);
C.prototype.constructor = C;